src/Routes.js

```diff
@@ -27,6 +27,9 @@ export const Routes = () => {
             { window.insights.chrome.isBeta() ?
                 <InsightsRoute exact path='/baselines' component={ BaselinesPage } /> : null
             }
+            { window.insights.chrome.isBeta() ?
+                <InsightsRoute path='/baselines/:id' component={ BaselinesPage } /> : null
```

Comment:
This feels like we are now overloading the BaselinesPage component and that if I go to this specific URL it should just render the BaselineEditPage

Reply:
That's fair. I'm not sure at this moment how much of a change this is going to require. Theoretically it should be an easy switch, but I'll need to work through it.